.husky/pre-commit

```diff
@@ -9,9 +9,28 @@ yarn lerna run \
   --stream \
   ;
 
+#
+#
+#
+
 if git diff --cached --name-only | grep --quiet ".k8s"
 then
   yarn k8s
   yarn k8s precommit
   git add .k8s
 fi
+
+#
+#
+#
+
+packagejsonfiles=$(
+  git diff --cached --name-only --diff-filter=ACM |
+  grep 'package\.json$' |
+  tr '\n' ' '
+)
+
+if [[ -z "$packagejsonfiles" ]]
+then
+  npx prettier-package-json --write "$packagejsonfiles"
```

Comment:
sure about npx in pre-commit ? nps is very slow and precommit should not be blocking

Reply:
I was just testing it.
I'm removing it.